relay/cmd/root.go

```diff
@@ -101,16 +105,19 @@ func waitForExitSignal() {
 func execute(cmd *cobra.Command, args []string) error {
 	err := cobraConfig.Validate()
 	if err != nil {
+		log.Debugf("invalid config: %s", err)
```

Comment:
Do we need these extra logs? Is the returned error not sufficient?

Reply:
The **SilenceErrors: true** so I think it is useful

Reply:
Good point, didn't see that. Why do we need it enabled?

Reply:
I would like to see logrus formatted lines only in the output. In other case, it will print out the error message itself without timestamp, line of the code, etc...

Reply:
Alright, let's fix it in the upstream Pr if possible